BoxContentSDK/BoxContentSDK/Requests/BOXCollaborationRemoveRequest.m

```diff
@@ -8,12 +8,6 @@
 #import "BOXAPIJSONOperation.h"
 #import "BOXContentSDKConstants.h"
 
-@interface BOXCollaborationRemoveRequest ()
-
-@property (nonatomic, readonly, strong) NSString *collaborationID;
```

Comment:
since we only need collaborationID as readonly in BOXCollaborationRemoveRequest, I didn't want to create a readwrite collaborationID. We can add that in if the need arises